### ISSUE TITLE
Module not found: Error: Cannot resolve module 'sweetalert' in 1.1.1

### ISSUE THREAD

**Issue by chiedojohn** [t4t5/sweetalert#459]
Hey there!
Since 1.1.1, I get the following error

```
Module not found: Error: Cannot resolve module 'sweetalert'
```

Any ideas?

**Comment by Martindelataille** [comment]
Same issue.

**Comment by antoniwan** [comment]
+1

**Comment by ingro** [comment]
The `main` field in `package.json` has been changed from *dist/sweetalert.min.js* to *lib/sweetalert.js* which is ignored by git.

**Comment by kcalliauw** [comment]
+1

**Comment by t4t5** [comment]
Fixed in 1.1.2, sorry 'bout that! :)

**Comment by developerdesigner18** [comment]
@t4t5, I am facing this issue. My sweetalert installed version is 2.1.2. So Do I need to decrease to 1.1.2 as you suggest?
